scripts/a2_collections

```diff
@@ -1,8 +1,21 @@
 #!/bin/bash
 
+# Prerequisites:
+#    jq preprocessor
+#       https://stedolan.github.io/jq/
+#    variable $TOK
+#       An admin token obtained from `get_admin_token` or `chef-automate admin-token`
+#    variable $TARGET_HOST
+#       A host running A2, typically `https://a2-dev.test`
+#
+# These works either inside or outside Habitat Studio.
+
+
+#############
+# authQuery
+#############
 # Purpose:
-#     Provide a very concise way to fetch collections of a2 auth entities at the API level.
-#     This works either inside or outside Habitat Studio.
+#     Provide a very concise way to fetch collections of a2 auth resources at the API level.
```

Comment:
Are "auth" resources the same as "IAM" resources? If so, we should be consistent with using one. Given that this is likely to only be used internally, it probably doesn't matter too much, but it gave me a moment's pause.

Reply:
I would think they mean the same; what would be an example of an auth resource that is not an IAM resource?